code-of-conduct.html

```diff
@@ -11,20 +11,20 @@
 	</HEAD>
 	<BODY>
 		<nav>
-					<div class="homeLogo">
-						<a href="https://www.realdevsquad.com/"><img src="img/logos/favicon.ico"  width="35px" height="auto"></a>
-					</div>
-          <input type="checkbox" id="nav-toggle">
-          <label for="nav-toggle" class="burger-menu">
-           <img src="img/logos/burger.jpg" width="35px" height="auto" alt="">
-          </label>
-           <ul class="menu">
-            <li><a href="index.html">Home</a></li>
-            <li class="active"><a href="code-of-conduct.html">Code of Conduct</a></li>
-            <li><a href="faq.html">FAQ</a></li>
-            <li><a href="discord.html">Discord</a></li>
-           </ul>
-        </nav>
+			<div class="homeLogo">
+				<a href="https://www.realdevsquad.com/"><img src="img/logos/favicon.ico"  width="35px" height="auto"></a>
+			</div>
```

Comment:
I was talking about the indentation on line number 14, 15 and 16 😄

Reply:
Replaced tab with 2 spaces.